### ISSUE TITLE
unofficial-udify, allennlp,  and transformers  conflicting dependencies

### ISSUE THREAD

**Issue by ranjita-naik** [PKSHATechnology-Research/camphr#142]
I'm trying to install udify on WSL as shown below. 

$ pip install unofficial-udify==0.3.0 en-udify@https://github.com/PKSHATechnology-Research/camphr_models/releases/download/0.7.0/en_udify-0.7.tar.gz

ERROR: Cannot install unofficial-udify and unofficial-udify==0.3.0 because these package versions have conflicting dependencies.

The conflict is caused by:
    unofficial-udify 0.3.0 depends on transformers<3.0.0 and >=2.3.0
    allennlp 1.3.0 depends on transformers<4.1 and >=4.0
    unofficial-udify 0.3.0 depends on transformers<3.0.0 and >=2.3.0
    allennlp 1.2.2 depends on transformers<3.6 and >=3.4
    unofficial-udify 0.3.0 depends on transformers<3.0.0 and >=2.3.0
    allennlp 1.2.1 depends on transformers<3.5 and >=3.1
    unofficial-udify 0.3.0 depends on transformers<3.0.0 and >=2.3.0
    allennlp 1.2.0 depends on transformers<3.5 and >=3.1
    unofficial-udify 0.3.0 depends on transformers<3.0.0 and >=2.3.0
    allennlp 1.1.0 depends on transformers<3.1 and >=3.0

Is this a known issue? Could you suggest a workaroudn please?

**Comment by ranjita-naik** [comment]
I also tried the following (note that I've removed unofficial-udify==0.3.0). Installation went through, but the prediction failed. 

$ pip install  en-udify@https://github.com/PKSHATechnology-Research/camphr_models/releases/download/0.7.0/en_udify-0.7.tar.gz

$ pip show unofficial-udify
Name: unofficial-udify
Version: 0.2.1
Summary:
Home-page: None
Author: Dan Kondratyuk
Author-email: None
License: None
Requires: conllu, allennlp, transformers
Required-by: camphr-allennlp


import spacy
nlp = spacy.load("en_udify")
doc = nlp("Udify is a BERT based dependency parser")


File "/home/nlp/.local/lib/python3.6/site-packages/allennlp/common/registrable.py", line 185, in resolve_class_name
    f"{name} is not a registered name for {cls.__name__}. "
allennlp.common.checks.ConfigurationError: udify_model is not a registered name for Model. You probably need to use the --include-package flag to load your custom code. Alternatively, you can specify your choices using fully-qualified paths, e.g. {"model": "my_module.models.MyModel"} in which case they will be automatically imported correctly.

**Comment by tamuhey** [comment]
Hi, thank you for bug reporting!
I've just published `camphr_allennlp==0.1.2`, which solves your problem.
Please uninstall all packages and re-install udify as follows:

```bash
$ pip install en-udify@https://github.com/PKSHATechnology-Research/camphr_models/releases/download/0.7.0/en_udify-0.7.tar.gz
```

**Comment by ranjita-naik** [comment]
Thanks for looking into this. I retried the installation in a new env.   Now it's failing with the following error.

(camphr_udify) >pip install en-udify@https://github.com/PKSHATechnology-Research/camphr_models/releases/download/0.7.0/en_udify-0.7.tar.gz

ERROR: Cannot install camphr-allennlp because these package versions have conflicting dependencies.

The conflict is caused by:
    allennlp 1.1.0 depends on torch<1.7.0 and >=1.6.0
    allennlp 1.0.0 depends on torch<1.6.0 and >=1.5.0

-Ranjita
